jbt-it/neomdb
# #135 client ips

- Implemented Frontend for Internal Project Page
- Dummy Object Inserted to display data

How to test:
- go to /internes-projekt/5 and check if all the data is displayed correctly
- check if the fields are editable and change a value
- check if the value is actually changed

Note: The api calls will be implemented latear
